pkg/testsuite/suite.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testsuite provides test environment to run E2E and integration tests..
+package testsuite
+
+import (
+	"context"
+	"testing"
+)
+
+const (
+	realmName       = "test-realm"
```

Comment:
Are these actually _prefixes_?

Reply:
Not really, these values overrides corresponding configs

Reply:
Hmm - but these need to be globally unique. How will that work?

Reply:
I think it's because in test suite, we will first check if realm exists. Anyway, make it a prefix and generate a random name every time.